core/jobs/src/main/java/com/cognifide/aet/job/common/utils/ParamsHelper.java

```diff
@@ -0,0 +1,160 @@
+/**
+ * Automated Exploratory Tests
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.aet.job.common.utils;
+
+
+import com.cognifide.aet.job.api.exceptions.ParametersException;
```

Comment:
Google style guide changed imports settings for IntelliJ in [one of the last updates](https://github.com/google/styleguide/commit/75c289f1d33836d1ff4bd94e6c9033673e320b58). It may occur that we have some formatted with the old and some with the new version.